### ISSUE TITLE
New logging clutters output

### ISSUE THREAD

**Issue by falexwolf** [scverse/scanpy#684]
The rehauled logging module tends to clutter output, see, for instance
```
import scanpy as sc
sc.settings.verbosity = 2
adata_sc = sc.datasets.pbmc68k_reduced()
sc.pp.neighbors(adata_sc)
```
which produces
![image](https://user-images.githubusercontent.com/16916678/59159145-a56bb300-8ac5-11e9-8371-170995e43dfa.png)

**Comment by falexwolf** [comment]
First step: removed logging of `asctime`, which we never had before: https://github.com/theislab/scanpy/commit/4650568cf61d8a654e64abd1ec0807e19423f1ff

**Comment by falexwolf** [comment]
Second step: reverted things that were logged at a level equal or higher than 4 to `debug`. a37efc71876f1cd9ace1165d7f774e390d30343d

The only thing that remains is to reformat the time output, which now displays many useless digits after the seconds comma [and fix all other places in which similar things happened].

**Comment by flying-sheep** [comment]
Thank you!

> step: removed logging of `asctime`, which we never had before

I took that from your logging code in the data science bowl notebook, but I agree: People should add `{asctime}` to all formatting themselves if they use scanpy in some non-interactive code. We should document how to do that.

> Second step: reverted things that were logged at a level equal or higher than 4 to `debug`.

Good! FYI: I changed the log levels because the initial version of the overhaul only allowed timing information in `info`.

> The only thing that remains is to reformat the time output, which now displays many useless digits after the seconds comma [and fix all other places in which similar things happened].

I’m on it. Seems like I missed that because testing code only uses full seconds, which prevents the milliseconds from showing up.

**Comment by flying-sheep** [comment]
fixed in 709bafb8ed600daf5f9ee995a0dc845ac1e7e605
